frontend/packages/kubevirt-plugin/src/types/vm-import/ovirt/ovirt-vm.ts

```diff
@@ -1,10 +1,11 @@
 export type OvirtNIC = {
   id: string;
-  netid: string;
   netname: string;
   name: string;
   mac: string;
   interface: string;
+  vnicid: string;
+  vnicnetname: string;
```

Comment:
haven't seen where `vnicnetname` is being used.
if only vnic-profile-id is used by the UI, perhaps netname and vnicnetname can be omitted ?

Reply:
`vnicbetname` is something requested from the provider: https://github.com/ManageIQ/manageiq-v2v-conversion_host/blob/master/kubevirt-vmware/pkg/controller/ovirtprovider/client.go#L292

Reply:
We are not using it in the UI, but it is good to have it defined in this type in case we will use/show it in the future

Reply:
why not to add it when needed? we could cleanup provider code as well.

Reply:
I don't mind either way